sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -0,0 +1,293 @@
+    includeQueryStatistics?: boolean;
+    qualifiedNames?: string[];
+    query: string;
+    serverTimeoutInSeconds?: number;
+    timespan?: string;
+    workspace: string;
+    workspaceIds?: string[];
+    workspaces?: string[];
+}
+
+// @public
+export type ColumnDataType = string;
+
+// @public
+export const CommonDurations: {
+    readonly last7Days: "P7D";
+    readonly last3Days: "P3D";
+    readonly last2Days: "P2D";
+    readonly lastDay: "P1D";
+    readonly lastHour: "PT1H";
+    readonly last4Hours: "PT4H";
+    readonly last24Hours: "P1D";
+    readonly last48Hours: "P2D";
+    readonly last30Minutes: "PT30M";
+    readonly last5Minutes: "PT5M";
+};
+
+// @public
+export interface ErrorDetail {
+    additionalProperties?: any;
+    code: string;
+    message: string;
+    resources?: string[];
+    target?: string;
+    value?: string;
+}
+
+// @public
+export interface ErrorInfo {
+    additionalProperties?: any;
+    code: string;
+    details?: ErrorDetail[];
+    innererror?: ErrorInfo;
+    message: string;
+}
+
+// @public
+export interface GetMetricDefinitionsOptions extends OperationOptions {
+    metricNamespace?: string;
+}
+
+// @public
+export interface GetMetricDefinitionsResponse {
+    definitions: MetricDefinition[];
+}
+
+// @public
+export interface GetMetricNamespacesOptions {
+    startTime?: string;
+}
+
+// @public
+export interface GetMetricNamespacesResponse {
+    namespaces: MetricNamespace[];
+}
+
+// @public
+export interface LocalizableString {
+    localizedValue?: string;
+    value: string;
+}
+
+// @public
+export class LogsClient {
+    constructor(tokenCredential: TokenCredential, options?: LogsClientOptions);
+    // (undocumented)
+    queryLogs(workspaceId: string, query: string, options?: QueryLogsOptions): Promise<QueryLogsResult>;
+    // (undocumented)
+    queryLogsBatch(batch: QueryLogsBatch, options?: QueryLogsBatchOptions): Promise<QueryLogsBatchResponse>;
+}
+
+// @public (undocumented)
+export interface LogsClientOptions extends PipelineOptions {
+    host?: string;
+}
+
+// @public
+export interface MetadataValue {
+    name?: LocalizableString;
+    value?: string;
+}
+
+// @public
+export interface Metric {
+    displayDescription: string;
+    errorCode?: string;
+    id: string;
+    name: LocalizableString;
+    timeseries: TimeSeriesElement[];
+    type: string;
+    unit: MetricUnit;
+}
+
+// @public
+export interface MetricAvailability {
+    retention?: string;
+    timeGrain?: string;
+}
+
+// @public
+export interface MetricColumn {
+    name?: string;
+    type?: ColumnDataType;
+}
+
+// @public
+export interface MetricDefinition {
+    category?: string;
+    dimensions?: LocalizableString_2[];
+    displayDescription?: string;
+    id?: string;
+    isDimensionRequired?: boolean;
+    metricAvailabilities?: MetricAvailability[];
+    // Warning: (ae-forgotten-export) The symbol "LocalizableString" needs to be exported by the entry point index.d.ts
```

Comment:
forgot exports

Reply:
I believe these are happening because of the multi-swagger-file issue with metrics (I can only export one of the multiple LocalizableString interfaces).

We want to stop exposing LocalizableString anyways so I'll have to restructure these models, which should eliminate this error. This is coming in the next few PRs for this package.

Recorded this in https://github.com/Azure/azure-sdk-for-js/issues/15296